Bio/SeqUtils/CodonUsage.py

```diff
@@ -53,6 +55,8 @@
     'GLU': ['GAG', 'GAA'],
     'TYR': ['TAT', 'TAC']}
 
+# DNA bases that can occupy each codon position
+CodonBases = {'A' : 0, 'C' : 0, 'G' : 0, 'T' : 0}
```

Comment:
Defining this as a module level variable does not seem ideal. There is a risk of the end user altering it etc.

I think it would be better to define these dictionaries where you do ``CodonBases.copy()`` within those methods.